multus/multus.go

```diff
@@ -558,9 +585,16 @@ func cmdAdd(args *skel.CmdArgs, exec invoke.Exec, kubeClient *k8s.ClientInfo) (c
 	cniArgs := os.Getenv("CNI_ARGS")
 	for idx, delegate := range n.Delegates {
 		ifName := getIfname(delegate, args.IfName, idx)
+		rt, cniDeviceInfoPath := types.CreateCNIRuntimeConf(args, k8sArgs, ifName, n.RuntimeConfig, delegate)
+		if cniDeviceInfoPath != "" {
+			err = nadutils.CopyDeviceInfoForCNIFromDP(cniDeviceInfoPath, delegate.ResourceName, delegate.DeviceID)
```

Comment:
@amorenoz There are several places in the PR where I ignore the errors coming back from utility calls for the DP and CNI files. Is there anyway to know for sure if a file should exist and take extra action? Or leave as is and handle file if it exists and move on otherwise.

Reply:
The thing is, everything is optional for now right?
From Multul's point of view:
- If DP file does not exist, we may want to log, but that 's not an error
- If CNI does not create a file, we may want to log, but is not an error either.

If DP does exist, we do the copy and then it is magically removed after the delegation-chain call (i.e: when we write the NetworkStatus Annotation), then yes, that's an error.